src/universal/conf/app.conf

```diff
@@ -1,5 +1,5 @@
 # This is the base configuration file for the Mantis ETC client.
```

Comment:
maybe is a good time to get rid of this file(`app.conf`). IMO is just a useless proxy.

Reply:
(disclaimer: probably out of scope)